src/string.cr

```diff
@@ -853,11 +853,13 @@ class String
     char_at(index) { yield }
   end
 
+  # Returns the `Char` at the given *index*, or raises `IndexError` if out of bounds.
```

Comment:
Minor stylistic thingy: The raised exceptions are typically not mentioned in the first sentence. Consider if this method raised more than one exception, it would be pretty convoluted. And from a semantic point, the purpose and primary behaviour of this method is to return a character. So it should be described as such.
It raises only in case of an error. That should be mentioned, but in a separate paragraph.